### PR TITLE
Feature/work from home

### PR DESCRIPTION
**Pull request by danlee8653**
https://trello.com/c/AOGKELRa/187-employers-add-remote-work-from-home-to-taxonomy-of-work-types-and-remove-australia-wide-from-taxonomy-of-locations

Modified the taxonomies selector to allow for filtering of archived taxonomies.

### CODE REVIEW

File: dist/redux-modules.cjs.js [hunk 1]
```diff
@@ -3571,14 +3555,22 @@ var taxonomies = function taxonomies(state) {
...
     if (Number.isInteger(a.get(orderBy))) {
       return a.get('id') - b.get('id');
     } else {
       return a.get(orderBy).localeCompare(b.get(orderBy));
     }
   });
+
+  if (withoutArchived) {
+    result = result.delete("16");
+    console.log(result);
```
Comment (IgorNadj) [inline dist/redux-modules.cjs.js]: Debug left in

File: dist/redux-modules.cjs.js [hunk 2]
```diff
@@ -3571,14 +3555,22 @@ var taxonomies = function taxonomies(state) {
...
+  var result = taxonomies(state).get(taxonomy, Immutable__default.Map()).toOrderedMap().sort(function (a, b) {
     if (Number.isInteger(a.get(orderBy))) {
       return a.get('id') - b.get('id');
     } else {
       return a.get(orderBy).localeCompare(b.get(orderBy));
     }
   });
+
+  if (withoutArchived) {
+    result = result.delete("16");
```
Comment (IgorNadj) [inline dist/redux-modules.cjs.js]: Can you filter by result.archived instead?

File: dist/redux-modules.cjs.js (hunk 1 above)
Comment (danlee8653) [reply dist/redux-modules.cjs.js]: Hang on, that's not the latest commit.

File: dist/redux-modules.cjs.js [hunk 3]
```diff
@@ -1228,22 +1228,6 @@ function get(object, path, defaultValue) {
 
 var get_1 = get;
 
-/**
```
Comment (IgorNadj) [inline dist/redux-modules.cjs.js]: Not clear to me why this was deleted? If it's just cleanup can you put in a separate commit please

File: dist/redux-modules.cjs.js (hunk 2 above)
Comment (danlee8653) [reply dist/redux-modules.cjs.js]: That's not the latest commit.

Comment (IgorNadj) [reply dist/redux-modules.cjs.js]: Ah I see, can you squash your commits please

File: dist/redux-modules.cjs.js (hunk 1 above)
Comment (IgorNadj) [reply dist/redux-modules.cjs.js]: 👍

File: dist/redux-modules.cjs.js (hunk 3 above)
Comment (danlee8653) [reply dist/redux-modules.cjs.js]: That's the Babel transpiled dist file, so you can ignore it.

Comment (IgorNadj) [reply dist/redux-modules.cjs.js]: Ah, nice, no worries. Yeah we should do that someday
